auction-server/src/opportunity/api.rs

```diff
@@ -161,10 +164,14 @@ pub struct GetOpportunitiesQueryParams {
     /// The permission key to filter the opportunities by. Used only in historical mode.
     #[param(example = "0xdeadbeef", value_type = Option< String >)]
     pub permission_key: Option<Bytes>,
-    /// The time to get the opportunities from. Used only in historical mode.
+    /// The time to get the opportunities from.
     #[param(example="2024-05-23T21:26:57.329954Z", value_type = Option<String>)]
     #[serde(default, with = "crate::serde::nullable_datetime")]
     pub from_time:      Option<OffsetDateTime>,
+    /// The maximum number of opportunities to return. Capped at 100.
+    #[param(example = "20", value_type = usize)]
```

Comment:
I think a better way to include the 100 parameter is using the maximum attribute in the param.
Even better we can use something like this to automate the validation:
https://github.com/Keats/validator